Source/WebCore/loader/archive/cf/LegacyWebArchive.cpp

```diff
@@ -567,12 +567,8 @@ RefPtr<LegacyWebArchive> LegacyWebArchive::create(const String& markupString, Lo
             if (auto subframeArchive = create(*childFrame->document(), WTFMove(frameFilter), mainFrameFileName)) {
                 auto subframeMainResource = subframeArchive->mainResource();
                 auto subframeMainResourceURL = subframeMainResource ? subframeMainResource->url() : URL { };
-                if (!subframeMainResourceURL.isEmpty()) {
-                    if (subframeMainResourceURL.isAboutSrcDoc() || subframeMainResourceURL.isAboutBlank() || subframeMainResourceURL.protocolIsData())
-                        uniqueSubresources.add(childFrame->frameID().toString(), subframeMainResource->fileName());
-                    else
-                        uniqueSubresources.add(subframeMainResourceURL.string(), subframeMainResource->fileName());
-                }
+                if (!subframeMainResourceURL.isEmpty())
```

Comment:
So we don't save iframe if it's about:blank / URL-less? Do we have a test for that?

Reply:
We have API test WebArchive.SaveResourcesIframe for "about:blank", where src is not set on iframe element and  subframeMainResourceURL is not empty in that case. I am not sure when the main resource url is empty, I just skip it since it's likely a meaningless key in map.